projects/packages/my-jetpack/_inc/components/stats-section/style.module.scss

```
@@ -26,7 +44,67 @@ $actions-size: 28px;
 	letter-spacing: -0.15px;
 }
 
-.cards-list {
+.chart-container {
+	padding: $padding-vertical $padding-horizontal 0;
+	position: relative;
+	box-sizing: border-box;
+
+	.chart-empty {
+		align-items: center;
+		bottom: 0;
+		display: flex;
+		justify-content: center;
+		left: 0;
+		padding: 0 50px;
+		position: absolute;
+		right: 0;
+		top: 0;
+		z-index: 2;
+	}
+
+	.empty-state-card {
+		border: 1px solid var(--jp-gray-5);
+		background-color: #ffffffd9;
+		border-radius: 4px;
+		display: flex;
+		margin: 0;
+		min-width: 320px;
+		padding: 24px;
+	}
+
+	.empty-state-card__icon {
+		align-self: center;
+		color: var(--color-neutral-80);
+		flex-shrink: 0;
+		margin-right: 20px;
```

RTL aware alternative is better
```suggestion
		margin-inline-end: 20px;
```